### ISSUE TITLE
Support for Forge Multipart(Microblocks)

### ISSUE THREAD

**Issue by kimfy** [squeek502/CreativeBlocks#10]
I'm not sure if this already works. I'm doing something wrong then, I've tried using the wildcard, example: "ForgeMicroblock:microblock:*" but nothing changes. Forge Multipart or Microblocks uses nbt data to store what material it is so might be that you aren't supporting nbt yet? I don't know, would be cool to see in a future update if it's possible.

**Comment by squeek502** [comment]
Will look into it

**Comment by squeek502** [comment]
Yeah, will have to add a lot of custom support for this. `GameRegistry.findBlock("ForgeMicroblock", "microblock")` returns `null` and `Block.getBlockFromItem(microblockItem)` returns `Blocks.air`.

**Comment by kimfy** [comment]
Note to self: Maybe change this from FMP to Chisels & Bits? I don't think FMP exists any longer even
Edit: [this](https://github.com/amadornes/MCMultiPart) seems to be the new API people will use for Microblocks in 1.9+.
